pages/search/index.tsx

```diff
@@ -92,8 +96,28 @@ export default function Search({
     )
   }
 
+  const searchResultsHeadingRef = useRef(null)
+  useEffect(() => {
+    // don't focus on "Displaying n results..." if the page is not done loading
+    if (isLoading) return
+    // keep focus on sort by selector if the last update to the query was a sort
+    if (isFreshSortByQuery) {
+      return
+    }
+    // otherwise, focus on "Displaying n results..."
+    searchResultsHeadingRef?.current?.focus()
+  }, [isLoading, isFreshSortByQuery])
+
+  const [headerText, setHeaderText] = useState(
+    getSearchResultsHeading(searchParams, totalResults)
+  )
+  useEffect(() => {
+    setHeaderText(getSearchResultsHeading(searchParams, totalResults))
+  }, [totalResults, searchParams])
+
+  const searchForm = <SearchForm aggregations={aggs} />
   return (
-    <SearchResultsAggregationsProvider value={aggs}>
```

Comment:
Why the update from context to prop? Not that it's a bad thing but just curious about the update. If you don't need `SearchResultsAggregationsProvider` then remove the function/file.

Reply:
i moved things around when i thought i'd have to pass refs down. i still think the refactor makes sense, but it might actually not work (i totally missed the search form on the home page 🤦🏻

Reply:
It looks like the home search does work though. So is the homepage going to be updated in the future with a simliar update?

Reply:
what similar update?

Reply:
Never mind. I confused this with the issue of the searchbar not rendering on the homepage.